_opensearch/remote.md

```diff
@@ -29,7 +29,7 @@ testClusters {
 }
 ```
 
-Segment replication must also be enabled in order to use remote-backed storage.
+Segment replication must also be enabled to use remote-backed storage.
```

Comment:
must be turned on. . .

Reply:
Use of enable was intentional since it's the same verb used in the property.

Reply:
Sorry by property I mean:

`systemProperty 'opensearch.experimental.feature.replication_type.enabled', 'true'`

Reply:
oh got it! makes sense. . .thanks